<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Sharing human data</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="sharing-human-data_files/libs/clipboard/clipboard.min.js"></script>
<script src="sharing-human-data_files/libs/quarto-html/quarto.js"></script>
<script src="sharing-human-data_files/libs/quarto-html/popper.min.js"></script>
<script src="sharing-human-data_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="sharing-human-data_files/libs/quarto-html/anchor.min.js"></script>
<link href="sharing-human-data_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="sharing-human-data_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="sharing-human-data_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="sharing-human-data_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="sharing-human-data_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#sharing-human-data" id="toc-sharing-human-data" class="nav-link active" data-scroll-target="#sharing-human-data">Sharing human data</a>
  <ul class="collapse">
  <li><a href="#sharing-non-personal-data" id="toc-sharing-non-personal-data" class="nav-link" data-scroll-target="#sharing-non-personal-data">Sharing non-personal data</a></li>
  <li><a href="#sharing-personal-data" id="toc-sharing-personal-data" class="nav-link" data-scroll-target="#sharing-personal-data">Sharing personal data</a>
  <ul class="collapse">
  <li><a href="#european-genome-phenome-archive" id="toc-european-genome-phenome-archive" class="nav-link" data-scroll-target="#european-genome-phenome-archive">European Genome-phenome Archive</a></li>
  <li><a href="#fega-sweden" id="toc-fega-sweden" class="nav-link" data-scroll-target="#fega-sweden">FEGA Sweden</a></li>
  <li><a href="#aida-data-hub" id="toc-aida-data-hub" class="nav-link" data-scroll-target="#aida-data-hub">AIDA Data Hub</a></li>
  </ul></li>
  <li><a href="#if-data-cannot-be-deposited-in-a-repository" id="toc-if-data-cannot-be-deposited-in-a-repository" class="nav-link" data-scroll-target="#if-data-cannot-be-deposited-in-a-repository">If data cannot be deposited in a repository</a>
  <ul class="collapse">
  <li><a href="#scilifelab-data-repository" id="toc-scilifelab-data-repository" class="nav-link" data-scroll-target="#scilifelab-data-repository">SciLifeLab Data Repository</a></li>
  <li><a href="#biostudies" id="toc-biostudies" class="nav-link" data-scroll-target="#biostudies">BioStudies</a></li>
  </ul></li>
  <li><a href="#how-can-scilifelab-help-you-sharing-human-data" id="toc-how-can-scilifelab-help-you-sharing-human-data" class="nav-link" data-scroll-target="#how-can-scilifelab-help-you-sharing-human-data">How can SciLifeLab help you sharing human data?</a>
  <ul class="collapse">
  <li><a href="#submission-support-for-fega-sweden" id="toc-submission-support-for-fega-sweden" class="nav-link" data-scroll-target="#submission-support-for-fega-sweden">Submission support for FEGA Sweden</a></li>
  <li><a href="#data-access-management-support" id="toc-data-access-management-support" class="nav-link" data-scroll-target="#data-access-management-support">Data access management support</a></li>
  </ul></li>
  <li><a href="#resources" id="toc-resources" class="nav-link" data-scroll-target="#resources">Resources</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Sharing human data</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="sharing-human-data" class="level1">
<h1>Sharing human data</h1>
<div class="alert alert-warning" role="alert">
<p><b><i>Disclaimer!</i></b> The PI, as well as everyone with access to sensitive personal data, is responsible for following current laws and regulations, and SciLifeLab will not assume legal responsibility for advice provided in these guidelines.</p>
</div>
<!-- Generella riktlinjer för publicering av human data. Vilka olika nivåer finns.  
The recommendations on publishing human data varies depending on the nature of the human data in question. The principle "As open as possible, as closed as necessary" is of great importance when it comes to publishing human data. -->
<p><a href="/topics/research-involving-human-data"><b>See more guidelines on research involving human data <i class="bi bi-arrow-right-square-fill"></i></b></a> <br><br></p>
<section id="sharing-non-personal-data" class="level2">
<h2 class="anchored" data-anchor-id="sharing-non-personal-data">Sharing non-personal data</h2>
<p>If your data cannot be linked to a living person, there is nothing in GDPR that prohibits you from sharing the data publicly. Data may however be sensitive under other regulations, so make sure that you take all relevant legal aspects into account. If you are dealing with human data, you should of course also think about the ethical and societal implications of sharing the data.</p>
<!--There are many repositories for sharing human-related data publicly. Here are a few examples:

European Nucleotide Archive (ENA)
: The [European Nucleotide Archive](https://www.ebi.ac.uk/ena/browser/home) is a public repository for nucleotide sequences hosted by the European Bioinformatics Institute (EMBL-EBI).

ArrayExpress
: [ArrayExpress](https://www.ebi.ac.uk/biostudies/arrayexpress) is a repository for functional genomics data hosted by the European Bioinformatics Institute (EMBL-EBI).

SciLifeLab Data Repository
: The [SciLifeLab Data Repository](https://figshare.scilifelab.se) is a an institutional repository hosted by SciLifeLab for depositing and describing data related to life science research. It is particularily useful for sharing files that are not suited for discipline-specific repositories. -->
<p><a href="/data-life-cycle/share/"><b>Learn more about data sharing and repositories <i class="bi bi-arrow-right-square-fill"></i></b></a> <br><br></p>
</section>
<section id="sharing-personal-data" class="level2">
<h2 class="anchored" data-anchor-id="sharing-personal-data">Sharing personal data</h2>
<p>Data that needs to be protected from unauthorised access may be deposited in a <em>controlled access repository</em>. In these repositories access will typically only be granted after a formal application procedure. When submitting any personal data for archiving and sharing in a controlled access repository, you must be sure that you have the legal right to do so.</p>
<p>Here are some repositories for sharing and archiving data under controlled access:</p>
<section id="european-genome-phenome-archive" class="level3">
<h3 class="anchored" data-anchor-id="european-genome-phenome-archive"><a href="https://ega-archive.org">European Genome-phenome Archive</a></h3>
<p>This is a service for sharing personally identifiable genetic and phenotypic data resulting from biomedical research projects. The repository is hosted by the European Bioinformatics Institute (EMBL-EBI) and the Centre for Genomic Regulation (CRG). Any data submitted to the repository is subject to controlled access, which means that access to the data only will be granted after a formal application procedure.</p>
</section>
<section id="fega-sweden" class="level3">
<h3 class="anchored" data-anchor-id="fega-sweden"><a href="https://fega.nbis.se/">FEGA Sweden</a></h3>
<p>This is a repository for storing and sharing personally identifiable genetic and phenotypic data in Sweden in a way that meets the requirements of the General Data Protection Regulation (GDPR). It is part of the <a href="https://ega-archive.org/federated">Federated European Genome-phenome Archive</a> (FEGA), which is a network consisting of several national nodes and the European Genome-phenome Archive (EGA). FEGA Sweden is currently transitioning into operational mode and researchers may express their interest in depositing data to the repository by filling in <a href="https://fega.nbis.se/submission/submission-request.html">a web form</a>.</p>
</section>
<section id="aida-data-hub" class="level3">
<h3 class="anchored" data-anchor-id="aida-data-hub"><a href="https://datahub.aida.scilifelab.se/">AIDA Data Hub</a></h3>
<p>This platform is part of the <a href="https://medtech4health.se/aida/">Analytic Imaging Diagnostics Arena (AIDA)</a> and NBIS/SciLifeLab. It provides Data Sharing, Policy Support, and Services for researchers in Swedish medical imaging AI, and facilitates sharing of FAIR medical imaging research data for Open Science. The <a href="https://datahub.aida.scilifelab.se/sharing/">AIDA Data Sharing Policy</a> is a comprehensive resource developed by ~50 AIDA partners in healthcare, industry, and academia, which includes legal analyses, best practices, and practical instructions for how to carry out ethically and legally correct sharing of sensitive personal data such as medical images in Sweden and similar countries. The <a href="https://datahub.aida.scilifelab.se/datasets/">AIDA Dataset Register</a> provides Digital Object Identifiers (DOI) and landing pages for the datasets that have been shared on the AIDA Data Hub.</p>
</section>
</section>
<section id="if-data-cannot-be-deposited-in-a-repository" class="level2">
<h2 class="anchored" data-anchor-id="if-data-cannot-be-deposited-in-a-repository">If data cannot be deposited in a repository</h2>
<p>It is not always possible to deposit personally identifiable data in a controlled access repository. In that case, you could register a <em>metadata record</em> for the data. This record will serve multiple purposes: First, it will enable unambiguous identification of the data. Second, it will provide a description of the data so that others can understand what the data is about. Third, the metadata record may contain instructions on how to get access to the data.</p>
<p>Here are a few repositories where you can register metadata records:</p>
<section id="scilifelab-data-repository" class="level3">
<h3 class="anchored" data-anchor-id="scilifelab-data-repository"><a href="https://figshare.scilifelab.se">SciLifeLab Data Repository</a></h3>
<p>SciLifeLab is hosting an institutional repository, SciLifeLab Data Repository, for publishing any kind of research-related data, e.g.&nbsp;documents, figures, or presentations. This repository is an instance of Figshare, which means that every item will get a unique DOI. You can read more in the <a href="https://www.scilifelab.se/data/repository">repository’s guidelines</a>. If you need any help connected to the SciLifeLab Data Repository, please <a href="../../contact/">contact us</a>!</p>
</section>
<section id="biostudies" class="level3">
<h3 class="anchored" data-anchor-id="biostudies"><a href="https://www.ebi.ac.uk/biostudies/">BioStudies</a></h3>
<p>This database contains descriptions of biological studies and is especially suited for linking a study to datasets in different repositories. However, it can also be used to describe sensitive data that has not been uploaded to a repository. BioStudies is hosted by the European Bioinformatics Institute (EMBL-EBI).</p>
</section>
</section>
<section id="how-can-scilifelab-help-you-sharing-human-data" class="level2">
<h2 class="anchored" data-anchor-id="how-can-scilifelab-help-you-sharing-human-data">How can SciLifeLab help you sharing human data?</h2>
<p>If you are a researcher at a Swedish academic institution working in the life sciences, you can get help from SciLifeLab Data Management support team.</p>
<section id="submission-support-for-fega-sweden" class="level3">
<h3 class="anchored" data-anchor-id="submission-support-for-fega-sweden">Submission support for FEGA Sweden</h3>
<p>FEGA Sweden is not yet in production, but you can get help with preparations for a future submission to the repository. The FEGA Sweden Helpdesk is part of the larger data management support team at SciLifeLab. Visit the <a href="https://fega.nbis.se">FEGA Sweden website</a> to learn more about how you can prepare a submission and how to get in contact with the FEGA Sweden Helpdesk.</p>
</section>
<section id="data-access-management-support" class="level3">
<h3 class="anchored" data-anchor-id="data-access-management-support">Data access management support</h3>
<p>If your data is located on the Bianca server at UPPMAX (or in the future deposited in FEGA Sweden), we can help with handling data access requests.</p>
<div class="alert alert-warning" role="alert">
<p>Until FEGA Sweden is ready for both storing and sharing data, we can only support data sharing with principal investigators that are eligible for setting up a project on Bianca server at UPPMAX.</p>
</div>
<p>To let us handle access requests to your data, you need to take the following steps:</p>
<ol type="1">
<li>Store the data in your own project on Bianca at UPPMAX.</li>
<li>Adjust the Data Access Agreement and Data Access request forms (available from <a href="mailto:datacentre@scilifelab.se">datacentre@scilifelab.se</a>) to suit your particular case.</li>
<li>Prepare a metadata record for the dataset in the SciLifeLab Data Repository. It is important to not upload the human data, but do provide as much descriptive metadata as possible to improve the findability. Set <a href="mailto:datacentre@scilifelab.se">datacentre@scilifelab.se</a> as “access request email” and your own email address as “contact email”.</li>
<li>In your manuscript add a section on Data Availability and make sure it includes information on where to go for requesting access to the data. You could for example write the following in your manuscript: &gt; The data is deposited on a secure Swedish server and has been assigned a DOI (XXX). Data access requests may be submitted to the Science for Life Laboratory Data Centre through the DOI link.</li>
<li>When a user applies for access to <a href="mailto:datacentre@scilifelab.se">datacentre@scilifelab.se</a>, the application will be vetted and you will be contacted by us. Then you can make a decision on access approved or denied. Ideally, you can (should) delegate this decision to an independent Data Access Committee (DAC).</li>
<li>When a user has been approved for access, SciLifeLab Data Centre will document the processing and arrange for the physical file access. Until FEGA Sweden is up and running, the physical file access will be that you, as a PI, copies the dataset from your Bianca project space, to the specified project space of the approved applicant. Because of NAISS user policy, only PIs at a Swedish academic institution can have a project at Bianca. So to share data with a collaborator outside Sweden, the PI in Sweden needs to have applied for a project at Bianca. Please see the <a href="https://www.uppmax.uu.se/support/user-guides/bianca-user-guide/">Bianca user guide</a> for information on how to do this.</li>
<li>SciLifeLab Data Centre will keep track of current and expired approvals.</li>
</ol>
<p>If you need any help connected to data submission, please <a href="../../contact/">contact us</a>!</p>
</section>
</section>
<section id="resources" class="level2">
<h2 class="anchored" data-anchor-id="resources">Resources</h2>
<p>Please find below resources concerning research involving human data in form of training, guidance and/or tools.</p>
<p></p>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>